doc/source/main_install.rst

```diff
@@ -239,3 +243,41 @@ bash-completion installation path, which you can discover as follows::
 
 See the `bash-completion FAQ <https://github.com/scop/bash-completion#faq>`_
 for more information.
+
+Zsh completions
+~~~~~~~~~~~~~~~~
+
+Zsh completions are provided by the ``_bst`` completion script, available online
+(`src/buildstream/data/zsh/_bst <https://raw.githubusercontent.com/apache/buildstream/master/src/buildstream/data/zsh/_bst>`_)
+and in your local Git clone at ``src/buildstream/data/zsh/_bst``.
+
+Copy the above file to the completions location for your Zsh framework:
+
+**Prezto**::
+
+    cp src/buildstream/data/zsh/_bst ~/.zprezto/modules/completion/external/src/_bst
+
+You may have to reset your zcompdump cache, if you have one, and then restart your shell::
+
+    rm ~/.zcompdump ${XDG_CACHE_HOME:-$HOME/.cache}/prezto/zcompdump
+
+**Oh My Zsh**::
+
+    mkdir $ZSH_CUSTOM/plugins/bst
+    cp src/buildstream/data/zsh/_bst $ZSH_CUSTOM/plugins/bst/_bst
+
+You must then add ``bst`` to your plugins array in ``~/.zshrc``::
+
+    plugins(
+      bst
+      ...
+    )
+
+**None**::
```

Comment:
This would be probably clearer as "No framework" or "Vanilla zsh". Also I don't know how people using zsh do it, but shouldn't this be the first one?

Reply:
You're right, I did word that a bit confusing. I might be incorrectly assuming that everyone uses a framework with Zsh, so I reworded that to remove that assumption.
Added one more commit to revise that (as well as rebasing to 1.95.4)! Hopefully that looks better!
Thanks for the review!